examples/blogs__e2e-snapshots/snapshots.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
       }
     }
   },
-  "__version": "11.1.0",
+  "__version": "11.2.0",
```

Comment:
I was running the tests from the release/12.0.0 branch on my local cypress repo (which still has 11.2.0 as the version number), rather than via the binary.

Fixed.